cmd/deployIntegrationArtifact.go

```diff
@@ -0,0 +1,98 @@
+package cmd
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/SAP/jenkins-library/pkg/command"
+	cpi "github.com/SAP/jenkins-library/pkg/cpi"
```

Comment:
It works like this, but `cpi` is anyway the default. I would like to suggest to remove the explicit  `cpi` here unless there is another reason for having it here which I don't know.